frontend/api/authors.js

```diff
@@ -10,11 +10,17 @@ export const useGetAuthors = () => {
       }
     }
   `;
-  const { loading, error, data } = useQuery(query);
+  const [refetch, { loading, error, data }] = useLazyQuery(query, {
+    fetchPolicy: "network-only",
+  });
   return {
     authorsLoading: loading,
     authorsError: error,
     authors: data?.getAuthors || [],
+    refetchAuthors: async () => {
+      const response = await refetch();
+      return response?.data?.getAuthors;
+    },
```

Comment:
I think you should really examine how Apollo works. If you are using the hooks correctly, you should never need to access the data directly like this. This is a reoccuring problem we see throughout the DU code base.

https://www.apollographql.com/docs/react/local-state/local-state-management/